searcher/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+
+	"github.com/pkg/errors"
+
+	"github.com/sleepypioneer/resource-searcher/searcher/analyse"
+	"github.com/sleepypioneer/resource-searcher/searcher/resource"
+)
+
+func (s *service) scrape(url string) (*resource.Resource, error) {
+	// read file for the moment
+	// in the future: call the python scraper
+	data, err := ioutil.ReadFile("./resource.json")
+	if err != nil {
+		return nil, err
+	}
+
+	res := &resource.Resource{}
+	if err := json.Unmarshal(data, res); err != nil {
+		return nil, errors.Wrap(err, "unmarshalling JSON")
+	}
+	return res, nil
+}
+
+type service struct {
```

Comment:
Currently not relevant so just to keep as an idea, we could use an interface

```go
type Scraper interface {
    Scrape(string) (*resource.Resource, error)
}
```

later on and have it implemented by both a `Service` and `testService` with different scraping logics (online vs. offline). That could simplify testing.

Reply:
I really like this and would like to already implement it, I will do it in a separate PR.